doc/content/reference/gateway-rtt/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Gateway RTT"
+description: "Gateway RTT"
+---
+
+{{% tts %}} records Round Trip Times (RTT) between the Gateway Server and Gateways which act as a measure of latency.
+
+This section explains how {{% tts %}} records and uses these values.
+
+<!--more-->
+
+## Background
+
+Since {{% tts %}} communicates with gateways over the internet, a latency of transmitted messages is expected. This latency is not an issue for uplinks, but for downlinks, since the Rx windows are at specific intervals, the {{% tts %}} needs to be aware of network latency.
+
+For UDP gateways, the RTT is the time difference between a downlink message and the corresponding valid Tx Acknowledgement message.
+
+For LoRa Basics Station gateways, the `RefTime` parameter that is sent on the Tx Confirmation message is used as the RTT. There are safeguards in place to make sure that a misbehaving gateway doesn't skew the RTT measurement. If the `RefTime` value is greater than 10 seconds, it is considered invalid.
+
+## Downlink Scheduling
+The Gateway Server uses RTT values to adjust downlink time when scheduling downlinks. {{% tts %}} by default stores a maximum of 20 recent RTT entries. Also, only those that were recorded within the the last 30 mins are considered. 
+
+When scheduling downlinks, if there are 5 or more RTT values that have been recorded in the last 30 mins, {{% tts %}} gets the 90th percentile value of these entries and uses that as the Round Trip Time. This value is then offset with either the gateway timestamp or the gateway absolute time (if the gateway has GPS) on the downlink that is scheduled.
+
+If there are are lesser than 5 RTT entries in the past 30 mins, the downlink is scheduled with offsetting the RTT.
```

Comment:
so in both cases (<5 entries and >=5 entries) there is an offset?

Reply:
Good one. This is a typo. If it's <5 entries, downlink is scheduled **without** an offset.